### PR TITLE
(PDB-1627) Benchmark tests + improved runinterval tests

### PR DESCRIPTION
**Pull request by mullr**

### CODE REVIEW

File: src/puppetlabs/puppetdb/cli/benchmark.clj [hunk 1]
```diff
@@ -387,12 +333,29 @@
...
       (let [run-interval (time/minutes (:runinterval options))
             rand-lastrun (fn [run-interval]
                            (jitter (time/minus (time/now) run-interval)
-                                   (time/in-seconds run-interval)))]
-        (->> hosts
-             (mapv #(agent (assoc % :lastrun (rand-lastrun run-interval))))
-             (world-loop base-url run-interval rand-perc))))))
+                                   (time/in-seconds run-interval)))
+            close-to-stop-chans (mapv #(run-simulated-host (assoc % :lastrun (rand-lastrun run-interval))
+                                                              run-interval
```
Comment (ajroetker) [inline src/puppetlabs/puppetdb/cli/benchmark.clj]: Is alignment ok here?

Comment (mullr) [inline src/puppetlabs/puppetdb/cli/benchmark.clj]: no, but I killed this code anyway

**Comment by wkalt** [comment]
taking a look at this now

**Comment by wkalt** [comment]
worked through the issues with this with @mullr today. I suspect the tests have some race conditions but he's on top of it -- once that's resolved I'm +1, this is a big improvement.

**Comment by mullr** [comment]
@wkalt fixed up the test

**Pull request merged**